providers/src/airflow/providers/fab/www/webpack.config.js

```diff
@@ -0,0 +1,270 @@
+/*!
```

Comment:
Copy of `airflow/www/webpack.config.js`. I removed all the assets specific to pages. These assets are no longer necessary.